### PR TITLE
Update mall.json

### PR DESCRIPTION
**Pull request by lkernan**
Add Westfield brand.

### CODE REVIEW

File: data/brands/shop/mall.json [hunk 1]
```diff
@@ -81,6 +81,8 @@
       "locationSet": {"include": ["au", "nz"]},
       "tags": {
         "brand": "Westfield",
+        "brand:wikidata": "Q18378979",
+        "brand:wikipedia": "en:Westfield Group",
```
Comment (1ec5) [inline data/brands/shop/mall.json:85]: This Wikipedia article and Wikidata item are about a former company. The two sites are littered with the remnants of former iterations of the company. To confuse things further, I created [this item](https://www.wikidata.org/wiki/Q106623253) to represent the brand per se, which can be added to both this NSI entry and the Euro-American one above.

Comment (lkernan) [reply data/brands/shop/mall.json:85]: Thanks.  I wasn't sure if that was an appropriate solution or not from the Wikidata perspective. I'll update the PR to use that instead.

**Comment by bhousel** [comment]
Thanks @lkernan & @1ec5 👍

**Comment by bhousel** [comment]
related #4958 and #2743

**Pull request merged**